src/components/Navbar.jsx

```diff
@@ -4,13 +4,15 @@ import {
   faMicrophone,
   faAngleLeft,
   faGear,
+  faMagnifyingGlass,
 } from '@fortawesome/free-solid-svg-icons';
 import { Link } from 'react-router-dom';
 import { screenStyle } from '../tailwindStyles';
 
 const goBack = <FontAwesomeIcon icon={faAngleLeft} />;
 const user = <FontAwesomeIcon icon={faMicrophone} />;
 const settings = <FontAwesomeIcon icon={faGear} />;
+const search = <FontAwesomeIcon icon={faMagnifyingGlass} />;
 
 function Navbar() {
   return (
```

Comment:
- [ ] As requested by the previous reviewer, Please make sure to add search functionalities where users can filter items by some parameters(Company name in your case). You just added the search icon but please also add the search functionalities as well.